tools/bridge/bridge/transfer_recorder.py

```diff
@@ -33,6 +39,28 @@ def __init__(self, minimum_balance: int) -> None:
 
         self.is_validator: Optional[bool] = None
         self.balance: Optional[int] = None
+        self.last_fetcher_reached_head_event: Dict[
+            ChainRole, FetcherReachedHeadEvent
+        ] = {}
+
+    def get_state_summary(self):
+        return {
+            "is-validator": self.is_validator,
+            "balance": str(self.balance or -1),
+            "num-transfer-events": len(self.transfer_events),
+            "num-scheduled-hashes": len(self.scheduled_hashes),
+            "num-completions": len(self.completion_hashes),
```

Comment:
Please really reconsider the use of kebab case. The problem is that in a lot of languages including Python and JavaScript, the minus is not a valid character, so you break the workflow to just convert it to an object and access it via `obj.var` because `obj.in-valid` is invalid. There is a reason the convention of json is camelCase. If not please use at least snake_case. 
I know you said it's a deliberate decision, but a bad deliberate decision is still a bad decision.

Reply:
I don't think there is a general convention to use camelCase for json keys (see https://stackoverflow.com/questions/5543490/json-naming-convention). 

If you want to convert it to an object where you can use dot access it would be easy to that by replacing '-' with '_', so I don't buy into your argument.

I can switch to using snake_case however.